### DISCUSSION TITLE
Query: whdload auto-config broken after recent commits git->preview ~ expected WIP glitch?

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1244] [General]
Tail of cmdline...   `strace ./amiberry --autoload lha/AddamsFamily_v1.3_0419.lha`

openat(AT_FDCWD, "lha/AddamsFamily_v1.3_0419.lha", O_RDONLY) = 19
newfstatat(19, "", {st_mode=S_IFREG|0644, st_size=748319, ...}, AT_EMPTY_PATH) = 0
mmap(NULL, 748319, PROT_READ|PROT_WRITE, MAP_PRIVATE, 19, 0) = 0x7f4b9113d000
close(19)                               = 0
munmap(0x7f4b9113d000, 748319)          = 0
write(3, "WHDBooter - Selected Slave: Adda"..., 48) = 48
--- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=0x38} ---
+++ killed by SIGSEGV (core dumped) +++
Segmentation fault (core dumped)

Also occurs at GUI->Quickstart->WHDLoad auto-config: -> Select, when clicking on a whdload.lha in dropdown ...ie; crashes before you click OK

TIA

**Comment by midwan** [comment]
It seems to work for me, are you on the latest commit or perhaps you're a few behind?

**Comment by giantclambake** [comment]
Latest as of now, still doing the same thing -- only whdload affected, everything else working.

edit: from gdb...

Thread 1 "amiberry" received signal SIGSEGV, Segmentation fault.
0x000000000068f572 in tinyxml2::XMLNode::FirstChild (this=0x0) at src/include/tinyxml2.h:769
769	        return _firstChild

**Reply by midwan** [reply]
Oh, I can recreate it with AddamsFamily - will fix

**Reply by midwan** [reply]
Fixed now

**Comment by giantclambake** [comment]
Yeah, that's sorted, thanks  ... now it's doing this;

Disregard, is fixed
